File: dataset/java2cpp/java/SUM_FACTORS_NUMBER.java
int f_gold ( int n ) {
  int result = 0 ;
  for ( int i = 2 ;
  i <= Math . sqrt ( n ) ;
  i ++ ) {
    if ( n % i == 0 ) {
      if ( i == ( n / i ) ) result += i ;
      else result += ( i + n / i ) ;
    }
  }
  return ( result + n + 1 ) ;
}

File: result/repair/transcoder/java2python/python_repair/SUM_FACTORS_NUMBER.py
def f_gold ( n ) :
    result = 0
    i=2 
    while i<=math.sqrt(n) :
        if n % i == 0 :
            if i == ( n / i ) :
                result += i
            else :
                result += ( i + n / i )
        i += 1
    return ( result + n + 1 )

